chpl/chpl-service/src/main/java/gov/healthit/chpl/changerequest/dao/ChangeRequestDAO.java

```diff
@@ -100,7 +100,7 @@ private ChangeRequestEntity getEntityById(Long id) throws EntityRetrievalExcepti
                 + "LEFT JOIN FETCH dev.contact "
                 + "LEFT JOIN FETCH dev.statusEvents statusEvents "
                 + "LEFT JOIN FETCH statusEvents.developerStatus "
-                + "LEFT JOIN FETCH dev.publicAttestations devAtt "
+                + "LEFT JOIN FETCH dev.attestations devAtt "
```

Comment:
This file will probably need to be revisited when OCD-3956 gets merged to staging. I added more methods that may need the same HQL change.